src/styles.css

```diff
@@ -0,0 +1,690 @@
+    background-color: transparent;
+    border-bottom: 1px solid #FFF;
+    margin-bottom: -1px;
+    padding-bottom: 33px;
+}
+.main_menu_bg .navbar-nav>li.active>a:hover {
+    color: #e84c3d;
+}
+/*home section*/
+.home {
+    background: url(../images/header-bg.jpg) no-repeat 100% 100%;
+    text-align: center;
+}
+.home-overlay-fluid {
+    background-color: rgba(0, 0, 0, 0.0);
+    width: 100%;
+    padding: 250px 0px;
+}
+.single_slider p {
+    max-width: 70%;
+    font-size: 3.5rem;
+    margin: 10px auto;
+    font-family: 'Georgia', serif;
+    font-style: italic;
+    line-height: 45px;
+    color: #fff;
+}
+.single_slider h2 {
+    color: #fff;
+    font-size: 6rem;
+    font-family: 'latobold';
+    margin-bottom: 50px; 
+}
+.main_slider_area {
+    text-align: center;
+    margin-top: 160px;
+    color: black;
+    background: transparent;
+}
+/*register section*/
+.login-register {
+    height: 120px;
+    width: 100%;
+    display: flex;
+    font-family: "Lato Medium";
+    font-size: 22px;
+    line-height: 19px;
+    background-color: rgb(33, 66, 74);
+}
+
+.login {
+    border-top: 120px solid rgb(33, 66, 74);
+    border-right: 25.55px solid transparent;
+    height: 0;
+    width: 50%;
+    position: absolute;
+}
+
+.register {
+    border-bottom: 120px solid rgb(33, 66, 74);
+    border-left: 25.55px solid transparent;
+    height: 0;
+    width: 50%;
+    margin-left: 50%;
+    position: absolute;
+}
+
+.register-text {
+    position: absolute;
+    margin-top: 52px;
+    margin-left: 255px;
+    color: white;
+}
+
+.login-text {
+    position: absolute;
+    margin-top: -71px;
+    margin-left: 470px;
+    color: white;
+}
+
+.register:hover{
+    border-bottom: 120px solid rgb(47, 172, 108);
+    border-left: 25.55px solid transparent;
+    cursor: pointer;
+}
+
+.login:hover {
+    border-top: 120px solid rgb(47, 172, 108);
+    border-right: 25px solid transparent;
+    cursor: pointer;
+}
+/*domain section*/
+.domain {
+    padding: 80px 0px;
+}
+.head_title {
+    margin-bottom: 40px;
+}
+.head_title h2 {
+    font-size: 2.5rem;
+    color: #383737;
+    font-family: 'latoheavy';
+}
+.head_title p {
+    font-size: 1.5rem ;
+    color: #a2a2a2;
+    font-family: 'latolight';
+}
+.input-group-addon {
+    padding: 6px 12px;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 1;
+    color: #a2a2a2;
+    text-align: center;
+    background-color: transparent;
+    border: 1px solid #ccc;
+    border-radius: 1px;
+}
+.input-group-lg>.form-control, .input-group-lg>.input-group-addon {
+    height: 46px;
+    padding: 10px 16px;
+    font-size: 18px;
+    line-height: 1.33;
+    border-radius: 2px;
+}
+select {
+    border: none;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 1;
+    background-color: transparent;
+    text-align: center;
+}
+.sercelist {
+    border: 1px solid #ddd;
+    padding: 20px;
+    display: inline-block;
+    width: 100%;
+    margin-top: 40px;
+}
+ .list-inline>li {
```

Comment:
Be careful with using the same selector as third party library has. You can always add your own style and write css for your own selectors